### DISCUSSION TITLE
Attribute error on wavelink.NodeReadyEventPayload

### DISCUSSION THREAD

**Discussion by T2theoby** [EvieePy/Wavelink#308] [Q&A]
Hi there. I'm having an issue using the on_wavelink_node_ready() function laid out in the documentation.  I'm defining the functions parameters the same as in the documentation:
`@commands.Cog.listener()
    async def on_wavelink_node_ready(self, payload: wavelink.NodeReadyEventPayload) -> None:
        logging.info(f"Wavelink Node connected: {payload.node!r} | Resumed: {payload.resumed}")
`
The `payload: wavelink.NodeReadyEventPayload` seems to throw back the error:

AttributeError: module 'wavelink' has no attribute 'NodeReadyEventPayload'

Not sure this is an error in my code. I'm using a mostly unedited version of a discord music bot found here:
https://github.com/AnasGrzor/Z4-Music-Discord-Music-Bot.git

Thanks.

**Comment by chillymosh** [comment]
@T2theoby Can you make sure you have updated your library and Wavelink is version 3.4.0
